packages/create-start/src/utils/runCmd.ts

```diff
@@ -0,0 +1,43 @@
+import spawn from 'cross-spawn'
+
+export async function runCmd(
```

Comment:
when do we need `runCmd` instead of `runPackageManagerCommand`?

Reply:
just for git?

Reply:
Will simplify this

Reply:
Pushed https://github.com/TanStack/router/pull/2920/commits/91fde4a4beed927d8bc040356ca81279368dcb60